Makefile

```diff
@@ -40,7 +40,7 @@ KONG_TEST_CONTAINER_TAG?=5000/kong-$(RESTY_IMAGE_BASE)-$(RESTY_IMAGE_TAG)
 KONG_TEST_IMAGE_NAME?=localhost:$(KONG_TEST_CONTAINER_TAG)
 
 # This logic should mirror the kong-build-tools equivalent
-KONG_VERSION?=`echo $(KONG_SOURCE_LOCATION)/kong-*.rockspec | sed 's,.*/,,' | cut -d- -f2`
```

Comment:
I do not think this change is compatible with Kong OSS (it doesn't have the plug-in `distribution/` directory. @hutchic can confirm.

Reply:
during build time we create `distribution/` if it doesn't exist and copy in the `.requirements` file. It was the easiest way to use the same build tool on two code bases. It gives us the ability to "hash this entire directory" as one of our cache keys for example and avoid `if this_code_base else that` logic everywhere

https://github.com/Kong/kong-build-tools/blob/216e225ebaffc65df3f56b120b0682118896f236/Makefile#L298-L306

Reply:
I added https://github.com/Kong/kong/pull/9002/commits/5e065b5268c8c8a106d6c8906df212b85138e321 to keep CE and EE in sync. Thanks for pointing this out gs!

If that commit looks good, would you mind approving that PR so I can get it merged?

Reply:
@tyler-ball Anything that's applicable to both EE & CE can just go into kong-build-tools. These changes will be easier to track if that grep script is a part of this PR.